cmd/root.go

```diff
@@ -71,10 +71,10 @@ func NewClient() *api.Client {
 }
 
 /*
-	Configuration for JWT can come from (in this order)
-	* Environment
-	* Configuration ~/.stormforger.toml, ./.stormforger.toml
-	* Command line flag
+Configuration for JWT can come from (in this order)
+* Environment
+* Configuration ~/.stormforger.toml, ./.stormforger.toml
+* Command line flag
```

Comment:
```suggestion
 * Configuration for JWT can come from (in this order)
 * * Environment
 * * Configuration ~/.stormforger.toml, ./.stormforger.toml
 * * Command line flag
```

Reply:
hmm, I blindly accepted what gofmt told me to do 🤷